abnf/odata-abnf-construction-rules.txt

```diff
@@ -966,7 +966,7 @@ SQUOTE-in-string = SQUOTE SQUOTE ; two consecutive single quotes represent one w
 
 dateValue = year "-" month "-" day
 
-dateTimeOffsetValue      = year "-" month "-" day "T" timeOfDayValue      ( "Z" / SIGN hour ":"   minute )
+dateTimeOffsetValue      = year "-" month "-" day "T" timeOfDayValue      ( "Z" / ( "+" / "-" ) hour ":"   minute )

```

Comment:
Same treatment for `decimalValue`, `durationValue`, `sbyteValue`, `int16Value`, `int32Value` and `int64Value`. All these must replace `SIGN` with `'+' / '-'` and they need a variant `...InUrl` where `SIGN` is used.

Reply:
@HeikoTheissen Do you want to open a Jira issue for that ABNF refactoring?

Reply:
https://issues.oasis-open.org/browse/ODATA-1606